standalone-metastore/metastore-server/src/test/resources/sql/postgres/upgrade-3.1.3000-to-4.0.0.postgres.sql

```diff
@@ -0,0 +1,77 @@
+-- The file has some overlapping with upgrade-3.2.0-to-4.0.0.postgres.sql
+SELECT 'Upgrading MetaStore schema from 3.1.3000 to 4.0.0';
```

Comment:
@zabetak qq: will this file be maintained  from now on parallel with upgrade-3.2.0-to-4.0.0.postgres.sql ? Is this needed for upstream?

Reply:
Yes, that's the idea for now; I couldn't think of a better alternative at the moment.

Reply:
How big is the differences between 3.2.0 and 3.1.3000? Is it not possible to manually apply changes to the schema to be 3.2.0 in the image and then use 3.2.0 -> 4.0.0 update path?

Reply:
Sure we can try do this but given that it might take some time I would rather leave it as a follow up. WDYT?

Reply:
You go with this, but I think we would need the follow up soon, this two upgrade path will cause some confusions :)

Reply:
I created HIVE-24480 for that purpose.